src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -138,10 +137,14 @@ public function get_grid_days( $date = null, $force = false ) {
 			$start = tribe_beginning_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
 			$end   = tribe_end_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
 
+			/*
+			 * We want events overlapping the current day, by more than 1 second.
+			 * This prevents events ending on the cutoff from showing up here.
+			 */
 			$day_query = tribe_events()->by_args( $repository_args )
-			                           ->where( 'date_overlaps', $start, $end )
-			                           ->per_page( $events_per_day )
-			                           ->order_by( $order_by, $order );
+									   ->where( 'date_overlaps', $start, $end, null, 2 )
+									   ->per_page( $events_per_day )
+									   ->order_by( $order_by, $order );
```

Comment:
:warning: Found precision alignment of 3 spaces.

<sub>hash: 9826d8d5f9658460317b7ca94712c8c9</sub>